### PR TITLE
136-fix-renamed Add Maintenance to Update Maintenance

### LINKED ISSUE #136: Change the NavigationTitle of the `EditEventDetailView`
## Describe the feature you want to improve
* The `EditEventDetailView` says `Add Maintenance` as the title but instead it should say `Update Maintenance` because we aren't adding something new, but updating values

## Describe alternatives you've considered
* N/A

## Screenshot

https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/22946902/6f34035c-5b84-45a4-8aa5-8aaee8acc2a2

### PR DESCRIPTION
**Pull request by AakifNadeem**
# What it Does
* Closes #136 
* Describe what your change does
Update Add Maintenance to Update Maintenance
Renamed Localizations as well.

# How I Tested
* Add a list of steps to show the functionality of your feature
For example:
* Run the application
* Tap the (+) button
* See this screen
etc...

# Notes
* Anything else that should be noted about how you implemented this feature?

# Screenshot
* Add a screenshot of your new feature! OR show a screen recording of it in action. (On the simulator press Cmd + R to record, and the stop button when done). **This video should be no longer than 30 seconds.**

### CODE REVIEW

**Comment by mikaelacaron** [comment]
@AakifNadeem do you know the languages you translated the new string to? If not, I'd prefer you leaving them empty and someone else can make a new PR updating the translations

**Comment by AakifNadeem** [comment]
> @AakifNadeem do you know the languages you translated the new string to? If not, I'd prefer you leaving them empty and someone else can make a new PR updating the translations

Its just quick google translation.

**Comment by mikaelacaron** [comment]
@AakifNadeem can you remove those then, I'd rather have real people making these translation for Hacktoberfest right now, thank you!

**Review by mikaelacaron — CHANGES_REQUESTED** [review]
Also be sure to fix this change, so that this isn't deleted in the xcscheme thanks!

**Comment by AakifNadeem** [comment]
@mikaelacaron I've removed the localization kindly check.

File: Basic-Car-Maintenance.xcodeproj/xcshareddata/xcschemes/Basic-Car-Maintenance-WidgetExtension.xcscheme [hunk 1]
```diff
@@ -90,7 +90,6 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES"
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance.xcodeproj/xcshareddata/xcschemes/Basic-Car-Maintenance-WidgetExtension.xcscheme:92]: Also please put back this line

**Review by mikaelacaron — CHANGES_REQUESTED** [review]
The wrong view was changed! See comment thanks. Also after you fix this, please click the re-review button, so I'm notified to take a look again
Re-review button

**Pull request closed without merging**